tests/integration_rust/snapshots/integration_rust__upgrade_tests__pypi_dependency_index_preserved_on_upgrade.snap

```diff
@@ -8,7 +8,7 @@ expression: redacted_content
         exclude-newer = "2025-05-19"
 
         [pypi-dependencies]
-click = { version = ">=8.2.0, <9", index = "https://pypi.tuna.tsinghua.edu.cn/simple" }
```

Comment:
This is immediately an interesting case that was previously "broken". There are whitespaces in front of all the keys. These were removed previously. The new implementation retains this.